ts/utils/__tests__/string.test.ts

```diff
@@ -27,6 +27,22 @@ describe("capitalize", () => {
   it("should return a string where each word has first char in uppercase-5", () => {
     expect(capitalize("   hello,world   ", ",")).toEqual("   Hello,World   ");
   });
+
+  it("should return a string where each word has the first char in uppercase even after an apostrophe", () => {
+    expect(capitalize("Capit'Alize")).toEqual("Capit'Alize");
+  });
+
+  it("should return a string where each word has the first char in uppercase even after an apostrophe-2", () => {
+    expect(capitalize("capit'alize")).toEqual("Capit'alize");
+  });
+
+  it("should return a string where each word has the first char in uppercase even after an apostrophe-3", () => {
+    expect(capitalize("capit'alize")).toEqual("Capit'alize");
+  });
```

Comment:
I noticed that the two test cases seem to be testing the same scenario (both with "capit'alize"). Would it make sense to keep just one of them to avoid redundancy?